webapp/src/Service/RejudgingService.php

```diff
@@ -361,11 +332,19 @@ public function calculateTodo(Rejudging $rejudging)
             ->select('COUNT(j)')
             ->andWhere('j.rejudging = :rejudging')
             ->andWhere('j.endtime IS NOT NULL')
+            ->andWhere('j.valid = 0')
```

Comment:
This is only relevant for auto apply rejudgings, right? I don't quite understand it's impact. Might be helpful to add a comment.

Reply:
Jep, added a comment.